chat-commands.js

```diff
@@ -31,7 +31,7 @@ exports.commands = {
 	'!version': true,
 	version: function (target, room, user) {
 		if (!this.runBroadcast()) return;
-		this.sendReplyBox("Server version: <b>" + Chat.package.version + "</b>");
+		this.sendReplyBox(`Server version: <b>${Chat.package.version}</b>.`);
```

Comment:
there doesnt need to be a period at the end of this, it's displaying the version so seeing something like ``1.2.3.`` is confusing for people